.prettierrc

```diff
@@ -1,5 +1,7 @@
 {
+  "printWidth": 90,
```

Comment:
any specific reason, besides preference for this: https://prettier.io/docs/en/options.html#print-width

Reply:
Nope. I'll remove.